README.md

```diff
@@ -322,6 +320,82 @@ The rules for `CanValidate` are as follows:
 
 # Alternatives
 
+## Making every field a pointer
+
+The recommended workaround to detect missing fields is to:
+
+1. Define every field as a pointer.
+2. Write a pass that checks every `nil` field and replaces it with an adequate default value.
+
+This means rewriting our example as follows:
+
+```go:
```

Comment:
The syntax highlighting [doesn't seem to work](https://github.com/pasqal-io/godasse/tree/yoric/doc#making-every-field-a-pointer), should we remove the `:` maybe ?